yilan_zhou/week_04/subway.rb

```diff
@@ -0,0 +1,53 @@
+#20/10/15 homework week4 Tue
+=begin
+Title: MTA Lab
+models a simple subway system.
+The program takes the line and stop that a user is 
+getting on at and the line and stop that user is getting off 
+at and prints the total number of stops for the trip.
+There are 3 subway lines:
+The N line has the following stops: 
+Times Square, 34th, 28th, 23rd, Union Square, and 8th
+The L line has the following stops: 
+8th, 6th, Union Square, 3rd, and 1st
+The 6 line has the following stops: 
+Grand Central, 33rd, 28th, 23rd, Union Square, and Astor Place.
+All 3 subway lines only intersect at Union Square, 
+=end
+
+$lines = {
```

Comment:
Good on using global variable here, but try to avoid them in future projects. It's ok to use them for small task like this one though.